traffic_ops/traffic_ops_golang/staticdnsentry/staticdnsentry.go

```diff
@@ -103,6 +104,11 @@ func (staticDNSEntry TOStaticDNSEntry) Validate() error {
 		addressErr = validation.Validate(staticDNSEntry.Address, validation.Required, is.IPv6)
 	case "CNAME_RECORD":
 		addressErr = validation.Validate(staticDNSEntry.Address, validation.Required, is.DNSName)
+		address := *staticDNSEntry.Address
+		lastChar := address[len(address)-1:]
```

Comment:
is it possible for `address` to have a length of 0? if so, this will fail, right?

Reply:
actually, even though it is required, it looks to me like it will still run this line of code:

`
lastChar := address[len(address)-1:]
`

Reply:
Updated the code to ensure the trailing period check takes place only if address length is greater than zero
![image](https://user-images.githubusercontent.com/22248619/89679579-a1484a80-d8ae-11ea-807a-84ba99e1c394.png)